qa/rest-compat-tests/src/main/java/org/elasticsearch/rest/compat/AbstractCompatRestTest.java

```diff
@@ -76,11 +77,17 @@ private static void mutateTestCandidate(ClientYamlTestCandidate testCandidate) {
             DoSection doSection = (DoSection) ds;
             //TODO: be more selective here
             doSection.setIgnoreWarnings(true);
-            //TODO: use the real header compatibility header
-            doSection.getApiCallSection().addHeaders(Collections.singletonMap("compatible-with", "v7"));
+
```

Comment:
note to other reviewers - the qa/rest-compat-tests is a temporary test fixture and currently runs the core REST rest tests from bwc:minor against a single current test cluster.  No need to review anything out of qa/rest-compat-tests since it will be going away in favor of per module/plugin tests. 

The (diminishing) number of failed core REST tests is what is guiding these early efforts.